mapper/pojo/src/main/java/org/hibernate/search/mapper/pojo/mapping/definition/annotation/AnnotationDefaultValues.java

```diff
@@ -14,6 +14,11 @@
 	 */
 	public static final String DO_NOT_INDEX_NULL = "__HibernateSearch_indexNullAs_default";
 
+	/**
+	 * Same default as scale attribute of JPA annotation Column when the precision attribute is not defined
```

Comment:
I think this should rather be an extreme value that realistically can't be used, so that we can detect when the user didn't specify anything.

For example, `Integer.MAX_VALUE`. Elasticsearch does not support scaling by a decimal scale higher than approximately ~300 (since it stores the scaling factor in a double), so this would be rather safe.

That way, you will be able to use the ORM metadata as a default, *and* users would still be able to force the decimal scale to 2 here even when they set it to 3 or 4 in the `@Column` annotation.

Other advantage: you would be able to detect when the scale wasn't configured at all, neither from the ORM metadata (because not using the ORM mapper) nor from the mapping (because `decimalScale` was not used). And you could throw an exception to warn the user of something dodgy. Because, let's be real, `2` might work for some use cases, but if you're using a BigDecimal you probably have specific needs and should configure the scale explicitly.

Note we do not need to be consistent with JPA, since ORM will always contribute something for every property and thus will never use defaults. So mentioning JPA in the javadoc is not necessary.